sass/README.md

```diff
@@ -1,3 +1,64 @@
-# Information
+# Overview
 
-As many of the services in the University are using Bootstrap, this styleguide is built ontop of Bootstrap's API also.
+Welcome to the University of Portsmouth living styleguide.
+
+Many services within the University are currently using [Bootstrap](https://github.com/twbs/bootstrap) and therefore this styleguide is built on top of it.
+
+This styleguide has been generated with [KSS](https://github.com/kss-node/kss-node) and is automatically built using [Travis](http://travisci.org) on commit to the master branch. You can view its associated GitHub repository [here](https://github.com/University-of-Portsmouth/styleguide).
+
+## How to use
+
+To get started please select a component from the sidebar.
+
+*TBC - Bower?*
```

Comment:
Yeah need to work out how it'll be consumable. Would prefer extending the SASS files, but realistically some people may need just a complete file.

I'd avoid Bower, npm is more suitable.